### PR TITLE
Switch to iterables for merging entry processor.

### PR DESCRIPTION
**Pull request by geekbeast**

### CODE REVIEW

**Comment by geekbeast** [comment]
Feel free to reject this pull request if breaks perf.

File: src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/SetStreamSerializers.java [hunk 1]
```diff
@@ -18,10 +19,17 @@
             c.accept( elem );
         }
     }
+    
+    public static <T> void serialize( ObjectDataOutput out, Iterable<T> elements, IoPerformingConsumer<T> c ) throws IOException {
+        out.writeInt( Iterables.size( elements ) ); //Iterables correctly does collections efficiently.
```
Comment (makosblade) [inline src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/SetStreamSerializers.java]: nit: can you move that comment up one line

**Comment by makosblade** [comment]
LGTM

**Comment by Bloodyhacker** [comment]
LGTM

File: src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/SetStreamSerializers.java (hunk 1 above)
Comment (geekbeast) [inline src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/SetStreamSerializers.java]: Yup!

**Pull request merged**